app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsFragment.kt

```diff
@@ -144,8 +121,44 @@ class OrderDetailsFragment : Fragment() {
                 rootView.orderDetailCoordinatorLayout.longSnackbar(it)
             })
 
-        orderDetailsViewModel.loadEvent(safeArgs.eventId)
-        orderDetailsViewModel.loadAttendeeDetails(safeArgs.orderId)
+        orderDetailsViewModel.event
+            .nonNull()
+            .observe(viewLifecycleOwner, Observer {
+                ordersRecyclerAdapter.setEvent(it)
+                Picasso.get()
+                    .load(it.originalImageUrl)
+                    .error(R.drawable.header)
+                    .placeholder(R.drawable.header)
+                    .into(rootView.backgroundImage)
+            })
+
+        orderDetailsViewModel.attendees
+            .nonNull()
+            .observe(viewLifecycleOwner, Observer {
+                if (it.isEmpty()) {
+                    Toast.makeText(context, getString(R.string.error_fetching_attendees), Toast.LENGTH_SHORT).show()
```

Comment:
Sanckbar

Reply:
> How you're revalidating attendees form API?

I don't think it is necessary to revalidate attendees from API as it is not possible to update attendees after order is made. If we need to then it is still possible as in https://github.com/fossasia/open-event-attendee-android/pull/2281 SwipeRefreshLayout is added to refresh order, which should also refresh attendees if both PRs got merged.